content/docs/2.7/providers/aws-eks.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "EKS Pod Identity Webhook for AWS"
+layout = "provider"
+weight = 3
```

Comment:
I suggest that we drop the weights in this and other providers' files, and let the providers be listed alphabetically.

Reply:
Done!